Lib/gzip.py

```diff
@@ -17,6 +17,14 @@
 
 READ, WRITE = 1, 2
 
+# OS field values as specified in RFC 1952
+OS_CODES = {
+    'linux': 3,
```

Comment:
There are many other supported Unix flavors: FreeBSD, NetBSD, OpenBSD, AIX, OpenIndiana, etc.

Reply:
I used C library (
http://git.savannah.gnu.org/cgit/gzip.git/tree/tailor.h#n113) as a
reference which sets 0x0b when it detects win32. From the looks of it a
value of 0x03 can be set for darwin, linux and aix.

Reply:
Fixed in 851c64b995b454e98d541236f48724bf2b59e3b2